integration-tests/operator/cmd/validate_instrumentation_vars.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func main() {
+
+	args := os.Args
+	namespace := args[1]
+	jsonPath := args[2]
+	success := verifyInstrumentationEnvVariables(namespace, jsonPath)
+	if !success {
+		fmt.Println("Instrumentation Annotation Injection Test: FAIL")
+		os.Exit(1)
+	} else {
+		fmt.Println("Instrumentation Annotation Injection Test: PASS")
+	}
+}
+
+func verifyInstrumentationEnvVariables(namespace string, jsonpath string) bool {
+
+	var args []string
+	if namespace != "default" {
```

Comment:
this if/else statement can be removed since we are passing in the namespace

so we can just do
```
args = []string{"get", "pods", "-n", namespace, "-l", "app=nginx", "-o=jsonpath='{.items[*].metadata.name}'"}
```